SDK_versioned_docs/version-3.0.0/widgets/swap-widget.mdx

```diff
@@ -0,0 +1,393 @@
+---
+id: swap-widget
+slug: swap-widget
+title: Getting Started with the Swap Widget
+sidebar_label: Getting Started
+sidebar_position: 1
+---
+
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Getting Started with the Swap Widget
+
+This guide walks you through the steps to embed the Uniswap Labs swap widget in your web3 decentralized application (dApp). The swap widget lets your users trade tokens directly with the Uniswap Protocol without leaving your dApp.
+
+Here’s a live preview of the swap widget. Please note that wallet connect is not supported here in the documentation. Your app is responsible for providing a method to connect a wallet.
+
+<iframe width="360" height="360" scrolling="no" src="https://widgets-uniswap.vercel.app/_renderer.html?_fixtureId=%7B%22path%22%3A%22src%2Flib%2Fcomponents%2FSwap%2FSwap.fixture.tsx%22%2C%22name%22%3Anull%7D"></iframe>
```

Comment:
Might want to add a TODO here to migrate this live preview to the actual npm package.

Reply:
added in plish trix